backend/initial_inject_to_db.py

```diff
@@ -1,287 +1,106 @@
 #!/usr/bin/env python3
 
 import os
-import re
 import glob
 import json
-import random
 import argparse
-from ccfatigue.models.database import Experiment, Test
+from ccfatigue.models.database import Experiment  # TODO  , Test
 from ccfatigue.services.database import __sync_url
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
 
 DATA_DIR = os.path.abspath(f"{__file__}/../../Data")
 REAL_EXPERIMENTS_TO_INJECT = [
-    f"{DATA_DIR}/TST_Khalooei_2021-10_FA",
+    f"{DATA_DIR}/preprocessed/TST_Khalooei_2021-10_FA",
```

Comment:
@sbancal from what I'm reading on https://github.com/orgs/EPFL-ENAC/projects/2/views/2 the only TST done are TST_Srinivasan_2021-10_FA and TST_Srinivasan_2021-10_QS .. Why do we have the TST_khalooei here ? Am I missing something ? Maybe I'm not understanding how it should work.

Reply:
```suggestion
    f"{DATA_DIR}/preprocessed/TST_Khalooei_2021-10_FA",
```

Reply:
You're right.
In fact, this hard-coded path should be replaced by something that browse `f"{DATA_DIR}/preprocessed/TST_*"`.

I changed this file because it's referencing the previous path (before we decided to create a *preprocessed* folder for the cleaned data.
But to make this script work again, it'll have to be completely adapted to the new data convention.
I plan to make these corrections the day the convention is finalized.